Source/NSubstitute.Acceptance.Specs/MultipleThreads.cs

```diff
@@ -112,6 +112,35 @@ public void Create_Delegate_Substitute_From_Many_Threads()
             Task.AwaitAll(tasks);
         }
 
+        [Test]
```

Comment:
Please add one more test to test that the last value is returned if you ask more values than configured.